src/rdb/tests/rdb_test.c

```diff
@@ -501,8 +509,14 @@ rdbt_test_tx(bool update, enum rdbt_membership_op memb_op, uint64_t user_key,
 		}
 	}
 
+	if (follower)
```

Comment:
An earlier version of this PR allows local TXs on started rdb objects and doesn't introduce rdb_storage objects. I had the following concerns about it.

1. A local TX can not yield between any two of its queries due to the concurrency concern. This may be an acceptable limitation, though I think may also be a little error-prone---not unacceptably so.
2. All API functions that accept only started rdb objects will segfault when given stopped rdb objects. It's better to enforce with compiler errors. (_For catastrophic recovery, we plan to use local TXs to read some values in an RDB replica before deciding whether we should reset the membership or not. It seems natural to do this without starting the DB replica._)

Hence the current version. I think if we want, in the future, we could allow a local TX with a started rdb object, requiring that it must not yield.

As to rdb_tx_query_pre, it makes sense and doesn't hurt to take the lock for local TXs too. I've made that change.